### DISCUSSION TITLE
2024年10月2日支付宝开通专业版无法升级

### DISCUSSION THREAD

**Discussion by RogerZhao1017** [getlantern/lantern-forum-cn#923] [Q&A]
![Screenshot_20241003_190832_com eg android AlipayGphone](https://github.com/user-attachments/assets/3fd98fcc-029a-4fae-bade-71abdb9e70f4)

支付后，跳转显示不在白名单
麻烦尽快处理下

**Comment by MaxMKMax** [comment]
@RogerZhao
您好. 谢谢您支持蓝灯. 已经帮您提交问题上去给负责人处理了.
请您也提供我账号邮箱.

**Reply by RogerZhao1017** [reply]
roger_zwl@qq.com

**Reply by RogerZhao1017** [reply]
@Mike777Khoo 还没有处理吗？

**Comment by RogerZhao1017** [comment]
***@***.***



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年10月3日(周四) 晚上11:35
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 2024年10月2日支付宝开通专业版无法升级 (Discussion #923)




 
@Rogerzhao
 您好. 谢谢您支持蓝灯. 已经帮您提交问题上去给负责人处理了.
 请您也提供我账号邮箱.

**Comment by RogerZhao1017** [comment]
@Mike777Khoo 请问这个什么时候能处理？都已经快一个月了开一个专业版这么难？

**Comment by MaxMKMax** [comment]
@RogerZhao1017 您好. 已经帮您提交问题上去处理了. 您这几天应该会收到邮件来登入专业版. 谢谢您的耐心等待. 我会让客服给您补偿三个月. 我试试看可以吗.
